conans/test/functional/toolchains/cmake/test_cmake_toolchain_win_clang.py

```diff
@@ -50,35 +57,203 @@ def build(self):
 
 
 @pytest.mark.tool_cmake
-@pytest.mark.tool_mingw64
-@pytest.mark.tool_clang(version="12")
+@pytest.mark.tool_clang(version="13")
+@pytest.mark.skipif(platform.system() != "Windows", reason="requires Win")
+class TestLLVMClang:
+    """ External LLVM/clang, with different CMake generators
+    This links always with the VS runtime, it is built-in
+    """
+
+    @pytest.mark.tool_mingw64
+    @pytest.mark.tool_visual_studio(version="17")
+    @pytest.mark.tool_clang(version="13")  # repeated, for priority over the mingw64 clang
+    @pytest.mark.parametrize("runtime", ["static", "dynamic"])
+    def test_clang_mingw(self, client, runtime):
+        """ compiling with an LLVM-clang installed, which uses by default the
+        VS runtime
+        """
+        client.run("create . pkg/0.1@ -pr=clang "
+                   "-s compiler.runtime_version=v143 "
+                   "-s compiler.runtime={}".format(runtime))
+        # clang compilations in Windows will use MinGW Makefiles by default
+        assert 'cmake -G "MinGW Makefiles"' in client.out
+        assert "GNU-like command-line" in client.out
+        assert "main __clang_major__13" in client.out
+        assert "main _MSC_VER193" in client.out
+        assert "main _MSVC_LANG2014" in client.out
+        assert "main _M_X64 defined" in client.out
```

Comment:
Not a huge deal since I'm assuming these tests will only run on the specific intersection of Windows with mingw64 (which I'm unsure if it works on ARM64 Windows). 

But we removed this type of hardcoded assumptions by adding passing `-s arch=x86_64` to the client run, or evaluating the expected value of the macro conditional to the default architecture (which we can query with `client.get_default_host_profile().settings['arch']`).

Reply:
I am leaving this as-is, as it is something that we cannot test at all. The moment we have win-arm machines and this fails, we can fix it.